src/main/java/com/labsynch/labseer/utils/SimpleUtil.java

```diff
@@ -135,7 +135,7 @@ public static String toAlphabetic(int i){
     		startIndex=endIndex;
     	}
     	for (String sqlClause : sqlCurveIdMap.keySet()){
-			String completeQueryString = queryString + sqlClause + " )";
+			String completeQueryString = queryString + sqlClause;
```

Comment:
Nice catch. This logic is a bit of a mess so it's good to have the tests confirm the fix.
I recall this being a hack to avoid Oracle's 1000 IN-clause limit. If we can find where this function is used it would be good to test cases with (assuming `PARAMETER_LIMIT=1000`) <1000, 1000-2000, >2000 entries just to make sure we haven't fixed one case and broken another.

Reply:
Thanks, found several uses of these functions which were now broken. This PR is massive now so opened this new one to solve this issue:

https://github.com/mcneilco/acas-roo-server/pull/318